openslides_backend/action/actions/mediafile/mixins.py

```diff
@@ -109,8 +109,14 @@ def check_title_parent_unique(
                 filter_ = And(filter_, Not(FilterOperator("id", "=", id_)))
             results = self.datastore.filter(self.model.collection, filter_, ["id"])
             if results:
+                parent_title = "NONE"
```

Comment:
Confusing for the user. Instead, write something like `File '{title}' already exists in the root folder` in case of no parent

Reply:
Use another error message in parent_id None case. This simplifies the logic.